test/plugin/test_out_google_cloud.rb

```diff
@@ -53,24 +53,22 @@ def test_client_401
 
   def test_partial_success
     setup_gce_metadata_stubs
-    {
-      APPLICATION_DEFAULT_CONFIG => 4.0,
```

Comment:
Are we no longer testing this without partial success enabled?

Reply:
This has been covered in our regular `test_prometheus_metrics` already, thus redundant.